### PR TITLE
[KO] Update korean translation

### PR DESCRIPTION
**Pull request by riseshia**
Changed:

- bump up version

Remained work:

Totally not translated below.

- debugging.md

@Dalgona, @malkoG, @devleoper, @marocchino  review please.

used diff: git diff dc9ab42..596722094 --word-diff -- lessons/

### CODE REVIEW

File: ko/lessons/basics/basics.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ iex> "foo\nbar"
 "foo\nbar"
 ```
 
-Elixir에는 이보다 더 복잡한 데이터 타입도 있습니다. 이런 부분은 컬렉션이나 함수를 다룰 때 조금 더 알아보도록 하겠습니다.
+Elixir에는 이보다 더 복잡한 데이터 타입도 있습니다. 이런 부분은 [컬렉션](../collections/)이나 [함수](../collections/)를 다룰 때 조금 더 알아보도록 하겠습니다.
```
Comment (marocchino) [inline ko/lessons/basics/basics.md]: `[함수](../collections/)` -> `[함수](../functions/)`

File: ko/lessons/basics/collections.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ iex> File.read("path/to/unknown/file")
 
 ## 키워드 리스트
 
-키워드(keyword)와 맵(map)은 Elixir의 연관 컬렉션입니다. Elixir에서 키워드 리스트란 첫번째 원소가 atom인 특별한 튜플들의 리스트입니다. 따라서 키워드 리스트는 리스트와 성능이 비슷합니다.
+키워드 리스트(keyword list)와 맵(map)은 Elixir의 연관 컬렉션입니다. Elixir에서 키워드 리스트란 첫번째 원소가 atom이고, 2개의 원소를 가지는 특별한 튜플들의 리스트입니다. 따라서 키워드 리스트는 리스트와 성능이 비슷합니다.
```
Comment (marocchino) [inline ko/lessons/basics/collections.md]: atom, 에텀 -> 애텀

File: ko/lessons/basics/enum.md [hunk 3]
```diff
@@ -131,7 +131,7 @@ iex> Enum.min([], fn -> :foo end)
 
 ### max
 
-`max/1`은 컬렉션 내의 `최대(max)` 값을 반환합니다.
+`max/1`은 컬렉션 내의 `최대(maximal)` 값을 반환합니다.
```
Comment (marocchino) [inline ko/lessons/basics/enum.md]: `최소(minimal)` -> 최소
`최대(maximal)` -> 최대

File: ko/lessons/basics/basics.md (hunk 1 above)
Comment (riseshia) [inline ko/lessons/basics/basics.md]: :+1:

File: ko/lessons/basics/collections.md (hunk 2 above)
Comment (riseshia) [inline ko/lessons/basics/collections.md]: :+1:

File: ko/lessons/basics/enum.md (hunk 3 above)
Comment (riseshia) [inline ko/lessons/basics/enum.md]: :+1:

File: ko/lessons/basics/enum.md [hunk 4]
```diff
@@ -115,7 +115,7 @@ iex> Enum.map([0, 1, 2, 3], fn(x) -> x - 1 end)
 
 ### min
 
-`min/1`은 컬렉션 내의 `최소(min)` 값을 찾습니다.
+`min/1`은 컬렉션 내의 `최소` 값을 찾습니다.
```
Comment (marocchino) [inline ko/lessons/basics/enum.md]: also need to remove backquote here.

Comment (marocchino) [inline ko/lessons/basics/enum.md]: `min/1`은 컬렉션 내의 최소 값을 찾습니다.

File: ko/lessons/basics/enum.md [hunk 5]
```diff
@@ -131,7 +131,7 @@ iex> Enum.min([], fn -> :foo end)
 
 ### max
 
-`max/1`은 컬렉션 내의 `최대(max)` 값을 반환합니다.
+`max/1`은 컬렉션 내의 `최대` 값을 반환합니다.
```
Comment (marocchino) [inline ko/lessons/basics/enum.md]: `max/1`은 컬렉션 내의 최대 값을 반환합니다.

**Comment by marocchino** [comment]
>_<;;;

**Comment by devleoper** [comment]
Oops..

**Comment by riseshia** [comment]
( ﾟдﾟ)

**Comment by devleoper** [comment]
@marocchino W…would you mind creating new PR and receiving credits from it? :trollface:

**Comment by riseshia** [comment]
in #1096

**Comment by devleoper** [comment]
🙇🙇🙇

**Pull request merged**